src/tests/system/tests/test_ldap.py

```diff
@@ -144,10 +144,10 @@ def test_ldap__password_change_new_password_does_not_meet_complexity_requirement
         "user1", "Secret123", "red_32"
     ), "Password should not have been able to be changed!"
 
-    assert (
-        "pam_sss(passwd:chauthtok): User info message: Password change failed."
-        in client.host.conn.run("journalctl").stdout
+    match = client.journald.is_match(
+        "pam_sss(passwd:chauthtok): User info message: Password change failed.", unit="sssd"
```

Comment:
I did not test the snippet I gave you, but it turned out it is not yet correct.

The message is not produced by sssd, so we need to leave the unit empty. Also the string is passed to --grep so it is a regular expression, therefore:

```python
    match = client.journald.is_match(
        r"pam_sss\(passwd:chauthtok\): User info message: Password change failed."
    )
```

There was also a bug in the journald util: https://github.com/next-actions/pytest-mh/pull/87 (merged)